### ISSUE TITLE
Not listed in Obsidian community plugins

### ISSUE THREAD

**Issue by lyndondrake** [omdenis/obsidian-id#1]
Hi,

I can't find this listed in Obsidian in the community plugins list, either under its project name or your GitHub username. Any suggestions on how to find it and enable it?

Best,
Lyndon

**Comment by omdenis** [comment]
Hi Lyndon,
You need to take the release version and manually copy into the plugin folder
YouVault\.obsidian\plugins 

And yes, I need to send to the public repository and register the plugin. Looks like it's valuable.
/Denis

**Comment by lyndondrake** [comment]
Yes, it'd be super useful. Where is that folder - is it part of Obsidian, or is it in the vault itself?

**Comment by omdenis** [comment]
It's inside the vault itself. All plugins are inside the vault. 

Before installing the plugin, please make  a backup of you vault.  This plugin is not reviewed by community and it was not included into the public repository yet. Maybe I will find some time to upload to them...

PS. But it works on my machine. And I like it :-)

**Comment by zhilinsu** [comment]
Hi Denis,  

I just want to say thank you! It's an extremely useful and helpful plugin, which is exactly what I need. Hope you can register the plugin, and I'm sure it can help loads of people. 

Zhilin

**Comment by utsecnet** [comment]
Hoping to see this registered in the plugin directory.

**Comment by omdenis** [comment]
Yeah, I’ve built up a few insanely cool plugins lately, and I think I’ll be publishing them on GitHub soon. I started with [avatar-voice](https://github.com/omdenis/avatar-voice)
.

I’ll be happy to share more once I’ve tested everything properly and made it stable.
